vanilla/todo.js

```diff
@@ -38,8 +45,10 @@ const createTaskNewRow = (table, taskInput) => {
 };
 
 const itemCheckboxChange = (event) => {
+    // i'm not sure path is encouraged or standard, and even if it was, it's too fragile
+    // what if you changed the html? this would break. maybe use "Element.closest" ?
     const row = event.path[2];
-    const taskTextCell = row.cells[1];
+    const taskTextCell = row.cells[1]; // this is fragile too
```

Comment:
yes, this is one of the places I really didn't like.
tried doing 
`event.target.parentNode()` like in the MDN but got an exception for parentNode not being a function...

Reply:
so yeah `closest` should work.